### PR TITLE
Chrome config bug

### PR DESCRIPTION
**Pull request by BMartinos**
This PR relates to issue: https://github.com/jembi/openhim-console/issues/249

Had to re-factor the way the config script is loaded. In chrome angular gets to the Api factory creation faster than in other browsers and by that time the config hasnt been set at certain times. 

Solution: Moved http request to get config details to be the first code to execute and upon receiving a result manually boostrapping the application to ensure config variables exist

### CODE REVIEW

**Comment by BMartinos** [comment]
Hey @rcrichton Do you mind just reviewing this PR for me when you get a chance. Thanks

File: app/scripts/app.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-angular
+var app = angular
   .module('openhimWebui2App', [
```
Comment (rcrichton) [inline app/scripts/app.js]: Maybe we should take this opurtunity to rename the module, its was pretty badly named (by me :) ) Maybe we can make this openhimConsoleApp?

**Comment by rcrichton** [comment]
Great, nicely done @BMartinos looks good to me. One comment above, but nothing to do with your code. :+1:

**Comment by BMartinos** [comment]
Hey @rcrichton i have made that module naming change and then a small change on app.js. I forgot i removed the title variable creation so just added it back in

**Comment by BMartinos** [comment]
Hey @rcrichton i have corrected the broken tests. I also removed some files which doesn't seem to be used or will be used.

**Comment by rcrichton** [comment]
Thanks @BMartinos this looks good :+1:

**Pull request merged**